package.json

```diff
@@ -20,38 +20,43 @@
     "type": "git",
     "url": "https://github.com/videojs/video.js.git"
   },
-  "main": "./dist/video-js/video.js",
+  "main": "src/js/video.js",
   "dependencies": {
     "videojs-swf": "4.5.4",
-    "vtt.js": "git+https://github.com/gkatsev/vtt.js.git#shim-build"
+    "vtt.js": "git+https://github.com/gkatsev/vtt.js.git#shim-build",
+    "global": "^4.3.0"
   },
   "devDependencies": {
-    "calcdeps": "~0.1.7",
+    "babelify": "^5.0.4",
+    "blanket": "^1.1.6",
+    "browserify-istanbul": "^0.2.1",
+    "browserify-versionify": "^1.0.4",
     "chg": "~0.2.0",
-    "contribflow": "~0.2.0",
-    "github": "~0.1.14",
     "grunt": "^0.4.4",
     "grunt-aws-s3": "^0.12.1",
-    "grunt-banner": "~0.2.0",
+    "grunt-browserify": "^3.5.0",
```

Comment:
I feel like we should just use browserify directly here instead of via grunt-browserify so we can use whatever version of browserify we want. But this shouldn't hold up this PR and can be changed anytime.